backfill-lending-markets.py

```diff
@@ -0,0 +1,49 @@
+import os
+from web3 import Web3
+from mev_inspect.compound_liquidations import fetch_all_underlying_markets, fetch_token_symbol
+from mev_inspect.crud.liquidations import write_ctoken_markets
+from mev_inspect.schemas.liquidations import CTokenMarket
+from mev_inspect.schemas.traces import Protocol
+from mev_inspect.db import get_inspect_session
+
+def store_markets():
```

Comment:
I think better for this to live in `cli.py` instead of its own script

`backfill.py` is really a bash script, but I did it in python because i'm bad at bash scripting lol